ui/package.json

```diff
@@ -18,31 +19,34 @@
     "@types/react-router-bootstrap": "^0.26.6",
     "@types/react-router-dom": "^5.3.3",
     "@types/yup": "^0.32.0",
-    "axios": "^1.6.5",
+    "axios": "0.27.2",
     "bootstrap": "^5.3.2",
     "eslint-config-prettier": "^9.1.0",
     "formik": "^2.4.5",
     "immer": "^10.0.3",
+    "openapicmd": "^2.4.0",
```

Comment:
is this the openapi generator that we are using? the docs in stack.md mention `openapi-codegen`, is that still used?